sova/awareness/providers/pr_status.py

```
@@ -0,0 +1,440 @@
+# ---------------------------------------------------------------------------
+
+
+def _get_author(pr: dict) -> str:
+    """Extract author login from a PR dict."""
+    author = pr.get("author")
+    if isinstance(author, dict):
+        return author.get("login", "")
+    return ""
+
+
+def _summarize_ci(rollup: list[dict]) -> str:
```

Ruff (0.16.2)

[warning] Too many branches (15 > 12)

(PLR0912)